jobs/Frontend/movieDatabase/src/components/moviesList.tsx

```diff
@@ -0,0 +1,41 @@
+import React, {FC, useEffect, useState} from "react";
+import './moviesList.less'
+import {MovieListItem} from "../types/movies";
+
+interface MoviesListProps {
+    movies?: MovieListItem[];
+    setSelectedMovieId?(id: number);
+
+}
+export const MoviesList: FC<MoviesListProps> = ({movies = [], setSelectedMovieId= (i: number) => {}}) => {

```

Comment:
Why default value used for props?
If there are no movies just check it and early return.